docs/runbooks/VirtApiRESTErrorsBurst.md

```diff
@@ -1,9 +1,8 @@
 # VirtApiRESTErrorsBurst
-<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
 
 ## Meaning
 
-More than 80% of REST calls have failed in the `virt-api` pods in the last 5 minutes.
+For at least the last 10 minutes, more than 80% of the REST calls to `virt-api` pods have been failing consistently.
```

Comment:
nit: `For at least the last 10 minutes` -> `For the past/last 10 minutes or longer, ...`

Reply:
@assafad  Updated